src/libsystemd/sd-bus/bus-common-errors.h

```diff
@@ -107,6 +107,7 @@
 #define BUS_ERROR_TOKEN_PROTECTED_AUTHENTICATION_PATH_NEEDED \
                                                "org.freedesktop.home1.TokenProtectedAuthenticationPathNeeded"
 #define BUS_ERROR_TOKEN_USER_PRESENCE_NEEDED   "org.freedesktop.home1.TokenUserPresenceNeeded"
+#define BUS_ERROR_TOKEN_USER_VERIFICATION_NEEDED "org.freedesktop.home1.TokenUserVerificationNeeded"
```

Comment:
doesn't matter, but above we break the line with `\` to keep the indentation coherent